tests/cephadm/test_cephadm_upgrade.py

```diff
@@ -80,6 +82,17 @@ def run(ceph_cluster, **kwargs) -> int:
         # Monitor upgrade status, till completion
         orch.monitor_upgrade_status()
 
+        if config.get("verify_cephadm_containers") and is_legacy_container_present(
+            ceph_cluster
+        ):
+            log.info(
+                "Checking cluster status to ensure that the legacy services are not being inferred"
+            )
+            if orch.cluster.check_health(
```

Comment:
Can we bring this outside of the if statement? It would be helpful to check the health of the system after the upgrade.

Reply:
The health is being checked after the upgrade as well, in the next few lines.

Reply:
In lines 96 to 100
        if config.get("verify_cluster_health"):
            if orch.cluster.check_health(
                rhbuild=config.get("rhbuild"), client=orch.installer
            ):
                raise UpgradeFailure("Cluster is in HEALTH_ERR state")